insights/client/__init__.py

```diff
@@ -503,29 +503,15 @@ def _f():
     return _f
 
 
-def die(msg=None, rc=None, retry=False, response=None):
-    """
-        Format and send the expected response to the parent/controlling client
-        process.
-
-        params:
-            msg (str): Content to be printed to stdout console
-            rc (int): return code to be used when exiting.
-                Unused if retry=True.  If the return code is not None, it
-                indicates that the parent process should stop executing phases
-                and exit.
-            retry (bool): True if the phase is considered to have failed and
-                the parent process should fall back to another egg.
-            response (str): Content to be passed back to the parent process,
-                potentially to be used as input other phases.
-    """
-    print json.dumps({
-        "message": msg,
-        "rc": rc,
-        "retry": retry,
-        "response": response
-    })
-    sys.exit(0)
+def die(msg=None, rc=0, retry=False, response=None, raw=False):
```

Comment:
I figure you aren't done here, but I don't see the point of having this function if it just calls sys.exit